Profiling: upgrade guide for self-hosted profiling agents (#3854)
elastic/observability-docs: docs/en/observability/profiling-upgrade.asciidoc

<titleabbrev>Upgrade</titleabbrev>
++++

> This section is specific to upgrading Universal Profiling when running it in {ecloud} or ECE.

Reviewer: ```suggestion
This section is specific to upgrading Universal Profiling on {ecloud} or ECE.

```

Author: Question here: is it ok to use both `{ecloud}` and `ECE`, or is there a placeholder for ECE? Or even, ECE docs are generated by using a different placeholder for `{ecloud}` so we should only have that in this sentence?

Reviewer: There is an {ece} attribute as well. The list of attributes is actually available [here](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc), and it's ok to use it here.

Author: Thanks, updating it

> For self-hosted installations, refer to <<profiling-upgrade-self-hosted,Upgrade Universal Profiling in self-hosted installations>>.

Reviewer: ```suggestion
For self-hosted installations, refer to <<profiling-self-managed-upgrade,Upgrade Universal Profiling in self-hosted installations>>.

```
Looks like this is why it's failing the build. I'll take a closer look at the rest of the doc here shortly.
